Makefile

```diff
@@ -13,7 +13,7 @@ export PATH := $(path_to_add):$(PATH)
 
 GO        := GO111MODULE=on go
 GOBUILD   := CGO_ENABLED=1 $(GO) build $(BUILD_FLAG)
-GOTEST    := CGO_ENABLED=1 $(GO) test -p 3
+GOTEST    := CGO_ENABLED=1 $(GO) test -p 4
```

Comment:
Because `Travis-ci` may OOM, the concurrent number was 3(https://github.com/pingcap/tidb/pull/4535).
From this [page](https://docs.travis-ci.com/user/reference/overview/#virtualization-environments), `Travis-ci` has 7.5GB RAM, we could make it large a little.

Reply:
If the cache does not work as you wish, you could delete it from this.
![image](https://user-images.githubusercontent.com/7782671/54009822-44bb9a00-41a7-11e9-833a-d2862bbb8d67.png)